access/advanced/add.md

```diff
@@ -36,7 +36,7 @@ Create a new **permission**.
 
 #### 📥 Parameters
 * `resource`<br>
-  The **resource profile** to grant this permission to.
```

Comment:
I don't think `AccessAdmin.add()` should be documented anymore since only `AccessAdmin.permit()` and `AccessAdmin.delete()` are supported.

Reply:
It must have gotten added back in when I went through merge conflicts. I agree. It has been deleted.